Dockerfile

```diff
@@ -108,13 +108,18 @@ ARG ROS_DEPS="apt-transport-https \
         libopenblas-dev \
         libpcap-dev \
         libpugixml-dev \
+        lttng-tools \
```

Comment:
Not a specific comment on this code change, but please create a GitHub Issue in the carma-platform repository that we can link this PR to. It would be great if you can include the deprecated warning that we are receiving in that issue as well.